src/docs/asciidoc/common/cloud-services/standard-user-workflow/presentation.adoc

```diff
@@ -1,2 +1,30 @@
 :relative-path: ../../../
-include::{docdir}/variables.adoc[]
\ No newline at end of file
+include::{docdir}/variables.adoc[]
+
+In a software application, we (almost) always need to manage users. The **Standard User Workflow** helps you to do this, in the most common use case. Currently, it only handles the registration of a user. In the following versions, this cloud service will help you to : 
+
+* Log in a user 
+* Log out a user
+* Manage a forgotten password
+* Manage permissions of users
+* etc... 
+
+NOTE: At this moment, the connection and the disconnection of a user is done through the ZetaPush SDK (We will see below how to do this).
+
+For us, the most common use case is when a user can create an account using a login and a password and needs to confirm his account via a link in an email.
+
+
+**Sign up : **
+
+image::./schemas/standardworkflowsignup.png[]
+<1> The user creates his account using a login, a password and an email. When the user submit the form, the account is created on the ZetaPush platform. In order to ensure that the user is not a bot, the account has to be confirmed before the user is authorized to login.
+<2> A email is sent to the user in order to confirm its account. The email contains the confirmation link. He needs to confirm his account before connection.
+<3> When the user clicks on the link in the email, the state of the account is changed on the ZetaPush platform and the user is redirected to a page of the application (specified by the developer). Afterthat, the user can connect to the application. 
```

Comment:
I think we don't need here too much technical explanation. Just functional explanation is enough.

~~When the user clicks on the link in the email, the state of the account is changed on the ZetaPush platform and the user is redirected to a page of the application (specified by the developer). Afterthat, the user can connect to the application.~~
When the user clicks on the link in the email, the user is authorized to login.